modules/custom/az_marketing_cloud/az_marketing_cloud.info.yml

```
@@ -0,0 +1,7 @@
+name: Quickstart for Marketing Cloud
+description: 'Provides News and Events HTML and related configuration for use within Marketing Cloud.'
+core_version_requirement: ^9 || ^10
+type: module
+package: 'The University of Arizona - Experimental'
+dependencies:
+  - az_news
```

Should we make it dependent on AZ Events now since it will eventually be dependent on it? 
```suggestion
  - az_news
  - az_events
```